src/Messaging/test/RabbitMQ.Test/Attributes/AsyncListenerTest.cs

```diff
@@ -0,0 +1,304 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Steeltoe.Common.Util;
+using Steeltoe.Messaging.Rabbit.Config;
+using Steeltoe.Messaging.Rabbit.Connection;
+using Steeltoe.Messaging.Rabbit.Core;
+using Steeltoe.Messaging.Rabbit.Exceptions;
+using Steeltoe.Messaging.Rabbit.Extensions;
+using Steeltoe.Messaging.Rabbit.Listener;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+using static Steeltoe.Messaging.Rabbit.Attributes.AsyncListenerTest;
+
+namespace Steeltoe.Messaging.Rabbit.Attributes
+{
+    [Trait("Category", "RequiresBroker")]
```

Comment:
Do we need to do anything around this trait for tests to work in DevOps?

Reply:
We shouldn't run any tests that RequiresBroker ... not sure how that is done?

Reply:
We can either replace `RequiresBroker` with `Integration` on these or (using parentheses where necessary) add or `Category!=RequiresBroker` to the test run filter in [the pipeline definition](https://github.com/SteeltoeOSS/Steeltoe/blob/master/azure-pipelines.yml#L25)

Reply:
Do we use Integration elsewhere for these kinds of "requires something else tests"? ... If so, we change to Integration to be more consistent!

Reply:
Yeah, we use that category on the config server integration tests

Reply:
Ok.. changed it to Integration